backend/open_webui/apps/webui/routers/auths.py

```diff
@@ -253,7 +263,7 @@ async def ldap_auth(request: Request, response: Response, form_data: LdapForm):
                 except Exception as err:
                     raise HTTPException(500, detail=ERROR_MESSAGES.DEFAULT(err))
 
-            user = Auths.authenticate_user(mail, password=str(form_data.password))
+            user = Auths.authenticate_user_by_trusted_header(mail)
```

Comment:
The `user` is authenticated before.
1. https://github.com/open-webui/open-webui/blob/2721c5939e55525d939fad6d0d117cfe222cc8eb/backend/open_webui/routers/auths.py#L244
2. https://github.com/open-webui/open-webui/blob/2721c5939e55525d939fad6d0d117cfe222cc8eb/backend/open_webui/routers/auths.py#L253-L255

Besides, there is no extra authentication in `/signup`, https://github.com/open-webui/open-webui/blob/2721c5939e55525d939fad6d0d117cfe222cc8eb/backend/open_webui/routers/auths.py#L438-L500

Is there any reason we do another db query here?

It is fixed in this PR which is closed with no reason, https://github.com/open-webui/open-webui/pull/6964/files#diff-26c1f42ed2225c0ee15cdc3aec215d94d4b1c8606a242754c73de2192a1ec5fbL222